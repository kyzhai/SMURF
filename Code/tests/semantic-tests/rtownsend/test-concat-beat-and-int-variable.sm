/* Testing concatenation of beats to acceptable ints through a variable */

x = [8,8,4]
y = [2,2,2,1..] ++ x
main = []
