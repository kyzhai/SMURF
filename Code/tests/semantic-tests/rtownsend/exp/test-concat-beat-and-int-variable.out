Program: 
	SVardef: 
		ID: x  :: [Int]

	[8, 8, 4]
	SVardef: 
		ID: y  :: [Beat]

	[2, 2, 2, 1..] ++ x
	SMain: []
Global Scope: 
	ID: main  :: []
	ID: x  :: [Int]
	ID: y  :: [Beat]
	ID: print x :: a -> a
	ID: random  :: Int

